### PR TITLE
fix(multilingual): do not use und in alias for i18n contents

### PR DESCRIPTION
**Pull request by dxvargas**
no additional commands needed

### CODE REVIEW

File: profiles/common/modules/features/nexteuropa_multilingual/nexteuropa_multilingual.module [hunk 1]
```diff
@@ -427,6 +430,16 @@ function nexteuropa_multilingual_path_update($path) {
  *     - language: The language of the alias.
  */
 function _nexteuropa_multilingual_path_und_sync($path) {
+
+  // Check if entity has a translation mechanism other than entity translation.
+  $node = menu_get_object("node", 1, $path['source']);
```
Comment (jfhovinne) [inline profiles/common/modules/features/nexteuropa_multilingual/nexteuropa_multilingual.module]: 1) OK, we are dealing with nodes only. Should perhaps be mentioned in the comments.
2) Are we sure $path['source'] is always set?

File: profiles/common/modules/features/nexteuropa_multilingual/nexteuropa_multilingual.module [hunk 2]
```diff
@@ -495,11 +508,18 @@ function nexteuropa_multilingual_path_delete($path) {
...
+ * If entity does not have a translation mechanism other than entity translation
+ * force all aliases to be saved as language neutral.
+ * All content translations will have a unique alias.
  */
 function nexteuropa_multilingual_pathauto_alias_alter(&$alias, array &$context) {
-  // Force all aliases to be saved as language neutral.
-  // All content translations will have a unique alias.
-  $context['language'] = LANGUAGE_NONE;
+
+  $translation_mechanism = variable_get('language_content_type_' . $context['type'], ENTITY_TRANSLATION_ENABLED);
```
Comment (jfhovinne) [inline profiles/common/modules/features/nexteuropa_multilingual/nexteuropa_multilingual.module]: Are we sure $context['type'] is always set?

**Comment by jfhovinne** [comment]
:+1:

File: profiles/common/modules/features/nexteuropa_multilingual/nexteuropa_multilingual.module (hunk 1 above)
Comment (gillesdeudon) [inline profiles/common/modules/features/nexteuropa_multilingual/nexteuropa_multilingual.module]: Concerning the $path['source'], on working to another ticket, I observed that it was not always set. I do not remember precisely in which conditions. I hesitate between content types that do not use "content translation" nor "Entity translation" or those that do not use revision.

**Comment by gillesdeudon** [comment]
:+1:

**Comment by jfhovinne** [comment]
:+1:

**Comment by gillesdeudon** [comment]
:+1:

**Pull request merged**